modules/sdk-core/src/bitgo/keychain/keychains.ts

```diff
@@ -427,4 +427,28 @@ export class Keychains implements IKeychains {
       throw new Error(`Error producing the output: ${errors}`);
     });
   }
+
+  /**
+   * Create keychain for ofc wallet using the password
+   * @param userPassword
+   * @returns
+   */
+  async createWalletKeychain(userPassword: string): Promise<Keychain> {
```

Comment:
```suggestion
  async createUserKeychain(userPassword: string): Promise<Keychain> {
```

Reply:
Its not technically a wallet keychain until you associate the key with the wallet. Keychains are standalone objects.

Reply:
Done